### PR TITLE
L2s part2 fixes

### PR DESCRIPTION
**Pull request by anselld**
Hi Hansie, I've completed the formatting, including the references and links, as well as sizing and centering of the figures. Please have a look and tell me if you're happy before I start with the technical editing.
Denise

### CODE REVIEW

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 1]
```diff
@@ -1,86 +1,132 @@
...
 
 #### Who does it?
 
-The Boston University provided a proof-of-concept and reference implementation alongside the white paper [[4]](https://github.com/BUSEC/TumbleBit). NTumbleBit [[5]](https://github.com/nTumbleBit/nTumbleBit) is being developed as a C# production implementation of the TumbleBit protocol that is currently being deployed by Stratis with their Breeze implementation ([[6]](https://stratisplatform.com/2017/07/17/breeze-tumblebit-server-experimental-release), [[7]](https://stratisplatform.com/2017/09/20/breeze-wallet-with-breeze-privacy-protocol-dev-update/)), currently at alpha/experimental release level in testnet.
+The Boston University provided a proof-of-concept and reference implementation alongside the white paper [[4]]. NTumbleBit [[5]] is being developed as a C# production implementation of the TumbleBit protocol that is currently being deployed by Stratis with their Breeze implementation [[6]], currently at alpha/experimental release level in testnet.
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
The Boston University provided a proof-of-concept and reference implementation alongside the white paper [[4]]. NTumbleBit [[5]] is being developed as a C# production implementation of the TumbleBit protocol that is currently being deployed by Stratis with their Breeze implementation ([[6]], [[7]]), currently at alpha/experimental release level in testnet.
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 2]
```diff
@@ -1,86 +1,132 @@
...
+Counterparty is NOT a block chain. Counterparty is a token protocol released in January 2014 that operates on Bitcoin. It has a fully functional Decentralized Exchange (DEX), as well as several hardcoded smart contracts defined that include contracts for difference and binary options ("bets"). To operate, Counterparty utilizes "embedded consensus", which means that a Counterparty transaction is created and embedded into a Bitcoin transaction, using encoding such as 1-of-3 multi-signature (multisig), Pay to Script Hash (P2SH) or Pay To Public Key Hash (P2PKH). Counterparty nodes, i.e. nodes that run both `bitcoind` and the `counterparty-server` applications, will receive Bitcoin transactions as normal (from `bitcoind`), and then `counterparty-server` will scan each, and decode and parse any embedded Counterparty transactions it finds. In effect, Counterparty is a ledger within the larger Bitcoin ledger, and the functioning of embedded consensus can be thought of similar to the fitting of one Russian stacking doll inside another.  ([[30]], [[31]])
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
Counterparty is NOT a block chain. Counterparty is a token protocol released in January 2014 that operates on Bitcoin. It has a fully functional Decentralized Exchange (DEX), as well as several hardcoded smart contracts defined that include contracts for difference and binary options ("bets"). To operate, Counterparty utilizes "embedded consensus", which means that a Counterparty transaction is created and embedded into a Bitcoin transaction, using encoding such as 1-of-3 multi-signature (multisig), Pay to Script Hash (P2SH) or Pay To Public Key Hash (P2PKH). Counterparty nodes, i.e. nodes that run both `bitcoind` and the `counterparty-server` applications, will receive Bitcoin transactions as normal (from `bitcoind`), and then `counterparty-server` will scan each, and decode and parse any embedded Counterparty transactions it finds. In effect, Counterparty is a ledger within the larger Bitcoin ledger, and the functioning of embedded consensus can be thought of similar to the fitting of one Russian stacking doll inside another.  ([[30]], [[31]], [[32]])
```

File: src/layer2scaling/more-landscape/landscape-update.md (hunk 1 above)
Comment (hansieodendaal) [reply src/layer2scaling/more-landscape/landscape-update.md]: Missed reference here, as shown above

File: src/layer2scaling/more-landscape/landscape-update.md (hunk 2 above)
Comment (hansieodendaal) [reply src/layer2scaling/more-landscape/landscape-update.md]: Missed reference here, as shown above

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 3]
```diff
@@ -93,82 +139,75 @@ COVAL is being developed with a primary purpose of moving value using “off-cha
...
 
-#### What is it?
-
-A 2-way peg (2WP) allows the "transfer" of BTC from the main Bitcoin block chain to a secondary block chain and vice-versa at a fixed rate by making use of an appropriate security protocol. The "transfer" actually involves BTC to be locked on the main Bitcoin block chain and unlocked/made available on the secondary block chain. The 2WP promise is concluded when an equivalent amount of tokens on the secondary block chain are locked (in the secondary block chain) so that the original bitcoins can be unlocked. ([[22]](https://www.rsk.co/blog/sidechains-drivechains-and-rsk-2-way-peg-design), [[28]](https://blockstream.com/technology/sidechains.pdf))
+*Sidechain: When the security protocol is implemented using Simplified Payment Verification (SPV) proofs - block chain transaction verification without downloading the entire block chain - the secondary block chain are referred to as a Sidechain.* [[22]]
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]: Missed reference here, as shown above

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 4]
```diff
@@ -223,67 +258,65 @@ Mimblewimble is being sited by Andrew Poelstra as being the ultimate *Scriptless
...
+- DAGlabs [[53]] (**Note:** This is the commercial development chapter.)
   - SPECTRE, PHANTOM
     - SPECTRE provides high throughput and fast confirmation times. Its DAG structure represents an abstract vote regarding the order between each pair of blocks, but this pairwise ordering may not be extendable to a full linear ordering due to possible Condorcet cycles.
     - PHANTOM provides a linear ordering over the blocks of the DAG and can support consensus regarding any general computation (smart contracts), which SPECTRE cannot. In order for a computation or contract to be processed correctly and consistently, the full order of events in the ledger is required, particularly the order of inputs to the contract. However, PHANTOM’s confirmation times are mush slower than those in SPECTRE.
 - Ethereum as the Ethash PoW algorithm that has been adapted from GHOST
-- [Dr. Bob McElrath](http://bob.mcelrath.org/resume/) ([[40]](https://scalingbitcoin.org/hongkong2015/presentations/DAY2/2_breaking_the_chain_1_mcelrath.pdf), [[41]](https://rawgit.com/mcelrath/braidcoin/master/Braid%2BExamples.html))
+- [Dr. Bob McElrath] ([[40]], [[41]])
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
- [Dr. Bob McElrath](http://bob.mcelrath.org/resume/) ([[40]], [[41]])
```

Comment (hansieodendaal) [reply src/layer2scaling/more-landscape/landscape-update.md]: A link to Dr. Bob McElrath's resume is provided here without making it a specific reference.

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 5]
```diff
@@ -1,86 +1,132 @@
...
...le projects built on top of Counterparty are [Age of Chains](https://www.ageofchains.com), [Age of Rust](http://spacepirate.io), [Augmentors](https://www.augmentorsgame.com/), [Authparty](http://authparty.io/), [Bitcorns](https://bitcorns.com/), [Blockfreight™](http://blockfreight.com/), [Blocksafe](http://www.blocksafefoundation.com), [BTCpaymarket.com](http://btcpaymarket.com), [CoinDaddy](http://coindaddy.io), [COVAL](https://coval.readme.io), [FoldingCoin](http://foldingcoin.net/), [FootballCoin](https://www.footballcoin.io/), [GetGems](http://getgems.org/#/), [IndieBoard](https://indiesquare.me/), [LTBCoin - Letstalkbitcoin.com](https://letstalkbitcoin.com/), [Mafia Wars](https://mafiawars.io/), [NVO](https://nvo.io/), [Proof of Visit](https://proofofvisit.com/), [Rarepepe.party](http://rarepepe.party), [SaruTobi Island](http://mandelduck.com/sarutobiisland/), [Spells of Genesis](http://www.spellsofgenesis.com), [Takara](https://mandelduck.com/#portfolio), [The Scarab Experiment](https://www.thescarabexperiment.org/), [Token.FM](https://token.fm/), [Tokenly](http://tokenly.com/), [TopCoin](https://topcoin.com/) and [XCP DEX](https://XCPDEX.COM). [[32]](https://counterparty.io)
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
The most notable projects built on top of Counterparty are [Age of Chains](https://www.ageofchains.com), [Age of Rust](http://spacepirate.io), [Augmentors](https://www.augmentorsgame.com/), [Authparty](http://authparty.io/), [Bitcorns](https://bitcorns.com/), [Blockfreight™](http://blockfreight.com/), 
[Blocksafe](http://www.blocksafefoundation.com), [BTCpaymarket.com](http://btcpaymarket.com), [CoinDaddy](http://coindaddy.io), [COVAL](https://coval.readme.io), [FoldingCoin](http://foldingcoin.net/), [FootballCoin](https://www.footballcoin.io/), [GetGems](http://getgems.org/#/), 
[IndieBoard](https://indiesquare.me/), [LTBCoin - Letstalkbitcoin.com](https://letstalkbitcoin.com/), [Mafia Wars](https://mafiawars.io/), [NVO](https://nvo.io/), [Proof of Visit](https://proofofvisit.com/), [Rarepepe.party](http://rarepepe.party), [SaruTobi Island](http://mandelduck.com/sarutobiisland/), 
[Spells of Genesis](http://www.spellsofgenesis.com), [Takara](https://mandelduck.com/#portfolio), [The Scarab Experiment](https://www.thescarabexperiment.org/), [Token.FM](https://token.fm/), [Tokenly](http://tokenly.com/), [TopCoin](https://topcoin.com/) and [XCP DEX](https://XCPDEX.COM). [[32]]
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 6]
```diff
@@ -93,82 +139,75 @@ COVAL is being developed with a primary purpose of moving value using “off-cha
...
-- <u>51% attacks:</u> 2WP secondary block chains are hugely dependent on merged mining. As such, mining power centralization and 51% attacks are a real threat, as demonstrated for Namecoin and Huntercoin (See [Merged Mining Introduction](https://github.com/tari-labs/tari-university/blob/master/merged-mining/merged-mining-scene/MergedMiningIntroduction.md)).
-- The DMMS provided by mining is not very secure for small systems, while the trust of the federation/notaries is riskier for large systems. [[28]](https://blockstream.com/technology/sidechains.pdf)
+- Security: Transferring BTC back into the main Bitcoin block chain is not secure enough and can be manipulated because Bitcoin does not support SPV from 2WP secondary block chains. [[21]]
+- 51% attacks: 2WP secondary block chains are hugely dependent on merged mining. As such, mining power centralization and 51% attacks are a real threat, as demonstrated for Namecoin and Huntercoin (See [Merged Mining Introduction](https://github.com/tari-labs/tari-university/blob/master/merged-mining/merged-mining-scene/MergedMiningIntroduction.md)).
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
- 51% attacks: 2WP secondary block chains are hugely dependent on merged mining. As such, mining power centralization and 51% attacks are a real threat, as demonstrated for Namecoin and Huntercoin (See [Merged Mining Introduction](../../merged-mining/merged-mining-scene/MergedMiningIntroduction.md)).
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 7]
```diff
@@ -299,136 +332,359 @@ DAG derivative protocols are not Layer 2 Scaling solutions, but offer significan
...
+[[21]] "Drivechain - The Simple Two Way Peg" [online].
+Available:  http://www.truthcoin.info/blog/drivechain. Date accessed: 2018-07-17.
+
+[21]: http://www.truthcoin.info/blog/drivechain
+"Drivechain - The Simple Two Way Peg"
+
+[[22]] "Sidechains, Drivechains, and RSK 2-Way Peg Design" [online].
+Available:  https://www.rsk.co/blog/sidechains-drivechains-and-rsk-2-way-peg-design or https://uploads.strikinglycdn.com/files/27311e59-0832-49b5-ab0e-2b0a73899561/Drivechains_Sidechains_and_Hybrid_2-way_peg_Designs_R9.pdf. Date accessed: 2018-07-18.
+
+[22]: https://www.rsk.co/blog/sidechains-drivechains-and-rsk-2-way-peg-design or https://uploads.strikinglycdn.com/files/27311e59-0832-49b5-ab0e-2b0a73899561/Drivechains_Sidechains_and_Hybrid_2-way_peg_Designs_R9.pdf
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
[22]: https://www.rsk.co/blog/sidechains-drivechains-and-rsk-2-way-peg-design
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 8]
```diff
@@ -299,136 +332,359 @@ DAG derivative protocols are not Layer 2 Scaling solutions, but offer significan
...
+[36]: https://eprint.iacr.org/2018/068.pdf
+"Simple Schnorr Multi-signatures with Applications to Bitcoin"
 
-[14] A Survey Of Second Layer Solutions For Blockchain Scaling  Part 1, https://www.ethnews.com/a-survey-of-second-layer-solutions-for-blockchain-scaling-part-1, https://www.ethnews.com/a-survey-of-second-layer-solutions-for-blockchain-scaling-part-1, Date accessed: 2018-07-16.
+[[37] A. Poelstra, 4 March 2017, "Scriptless Scripts" [online].
+Available:  https://download.wpsoftware.net/bitcoin/wizardry/mw-slides/2017-03-mit-bitcoin-expo/slides.pdf. Date accessed: 2018-07-24.
 
-[15] Second-Layer Scaling, https://lunyr.com/article/Second-Layer_Scaling, Date accessed: 2018-07-16.
+[37]: https://download.wpsoftware.net/bitcoin/wizardry/mw-slides/2017-03-mit-bitcoin-expo/slides.pdf
+Scriptless Scripts
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
"Scriptless Scripts"
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 9]
```diff
@@ -299,136 +332,359 @@ DAG derivative protocols are not Layer 2 Scaling solutions, but offer significan
...
-[12] Bitcoin Privacy is a Breeze: TumbleBit Successfully Integrated Into Breeze, https://stratisplatform.com/2017/08/10/bitcoin-privacy-tumblebit-integrated-into-breeze, Date accessed: 2018-07-13.
+[[36]] G. Maxwell et al, "Simple Schnorr Multi-signatures with Applications to Bitcoin" [online].
+Available: 20 May 2018, https://eprint.iacr.org/2018/068.pdf. Date accessed: 2018-07-24.
 
-[13] TumbleBit Wallet Reaches One Step Forward, https://www.bitcoinmarketinsider.com/tumblebit-wallet-reaches-one-step-forward, Date accessed: 2018-07-13.
+[36]: https://eprint.iacr.org/2018/068.pdf
+"Simple Schnorr Multi-signatures with Applications to Bitcoin"
 
-[14] A Survey Of Second Layer Solutions For Blockchain Scaling  Part 1, https://www.ethnews.com/a-survey-of-second-layer-solutions-for-blockchain-scaling-part-1, https://www.ethnews.com/a-survey-of-second-layer-solutions-for-blockchain-scaling-part-1, Date accessed: 2018-07-16.
+[[37] A. Poelstra, 4 March 2017, "Scriptless Scripts" [online].
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
[[37]] A. Poelstra, 4 March 2017, "Scriptless Scripts" [online].
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 10]
```diff
@@ -299,136 +332,359 @@ DAG derivative protocols are not Layer 2 Scaling solutions, but offer significan
...
+"TumbleBit: An Untrusted Bitcoin-compatible Anonymous Payment Hub"
+
+[[2]] E. Heilman et al., "TumbleBit: An Untrusted Bitcoin-compatible Anonymous Payment Hub" [online].
+Available: https://eprint.iacr.org/2016/575.pdf. Date accessed: 2018-07-08.
+
+[2]: https://eprint.iacr.org/2016/575.pdf
+"TumbleBit: An Untrusted Bitcoin-compatible Anonymous Payment Hub"
+
+[[3]] "Anonymous Transactions Coming to Stratis" [online].
+Available: [https://medium.com/@Stratisplatform/anonymous-transactions-coming-to-stratis-fced3f5abc2e](https://medium.com/@Stratisplatform/anonymous-transactions-coming-to-stratis-fced3f5abc2e). Date accessed: 2018-07-08.
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]:
```suggestion
Available: https://medium.com/@Stratisplatform/anonymous-transactions-coming-to-stratis-fced3f5abc2e. Date accessed: 2018-07-08.
```

File: src/layer2scaling/more-landscape/landscape-update.md [hunk 11]
```diff
@@ -299,136 +332,359 @@ DAG derivative protocols are not Layer 2 Scaling solutions, but offer significan
...
+"Counterparty Documentation: Protocol Specification"
 
-[56] Counterparty Documentation: Frequently Asked Questions, https://counterparty.io/docs/faq, Date accessed: 2018-09-14.
+[[58]] "Counterparty News: Why Proof-of-Burn, March 23, 2014" [online]. Available: https://counterparty.io/news/why-proof-of-burn. Date accessed: 2018-09-14.
 
-[57] Counterparty Documentation: Protocol Specification, https://counterparty.io/docs/protocol_specification, Date accessed: 2018-09-14.
+[58]: https://counterparty.io/news/why-proof-of-burn
+"Counterparty News: Why Proof-of-Burn"
 
-[58] Counterparty News:  Why Proof-of-Burn, March 23, 2014 , https://counterparty.io/news/why-proof-of-burn, Date accessed: 2018-09-14.
```
Comment (hansieodendaal) [inline src/layer2scaling/more-landscape/landscape-update.md]: Add to **Contributors** list: 

`- [https://github.com/anselld](https://github.com/anselld)`

File: src/layer2scaling/more-landscape/landscape-update.md (hunk 3 above)
Comment (anselld) [reply src/layer2scaling/more-landscape/landscape-update.md]: Thanks.

**Comment by anselld** [comment]
Thanks for the review - pulled the changes and have started with editing.

**Comment by anselld** [comment]
Started with technical editing. Will let you know when ready for review.

**Comment by anselld** [comment]
Hansie, only two questions please:
- [28] was not available online; please provide.
- The text reference to Dagger-Hashimoto did not have a corresponding reference number. Should it have?

**Comment by hansieodendaal** [comment]
[28] = https://blockstream.com/sidechains.pdf
Dagger reference = http://www.hashcash.org/papers/dagger.html
Hashimoto reference = https://web.archive.org/web/20170810043640/https://pdfs.semanticscholar.org/3b23/7cc60c1b9650e260318d33bec471b8202d5e.pdf

So Dagger-Hashimoto can have two references: Dagger-Hashimoto ([?], [?])

**Comment by anselld** [comment]
> [28] = https://blockstream.com/sidechains.pdf
> Dagger reference = http://www.hashcash.org/papers/dagger.html
> Hashimoto reference = https://web.archive.org/web/20170810043640/https://pdfs.semanticscholar.org/3b23/7cc60c1b9650e260318d33bec471b8202d5e.pdf
> 
> So Dagger-Hashimoto can have two references: Dagger-Hashimoto ([?], [?])

Thanks, will insert the two references, as well as the new link to [28].

**Comment by anselld** [comment]
Last two references required, please.

**Comment by hansieodendaal** [comment]
Sorry, I missed some things with the editing here:

1. Lightning Network must be capitalized (it is a system)

1. Let us not use the American way of including commas and full stops within quotes. We can note this in the style guide with a next pull request.

1. We need to use lower case for image extensions in the Markdown documents as well as with the actual file names itself as sometimes having those in upper case pose problems in GitHub, mdBook and GitPitch.

1. Time of writing any of these reports are generally when most references were accessed.

**Comment by anselld** [comment]
Hansie, with reference to your numbering above:

1. The term "lightning network" appears as a direct quote from [3]. I don't recommend that it should be changed here. The word 'lightning' is used elsewhere in the text (one occurrence), and I changed it to "Lightning".

2. Changed this report as suggested. 

3. To clarify, as an example, is the filename in title case just before line 80 correct, or should it be tumblebitoverview.png? Would it be helpful to mention this in the Style Guide?

4. Thanks - noted.

5. Have done so.

6. Have done so.

Another question, please: Is there any way of stopping the date in [57] and [58] from being split over two lines (like a non-breaking hyphen)?

**Comment by hansieodendaal** [comment]
Thanks @anselld, good job, only added some last minute changes and updated TOC links.

**Pull request merged**